docs/SETUP.md

```diff
@@ -52,9 +52,9 @@ $ docker run -it splunk/splunk:latest create-defaults > default.yml
 Alternatively, you can download the example `default.yml` supplied [here](advanced/default.yml.spec.md#sample).
 
 3. Define a few key variables in your `default.yml`:
-* `splunk.role`: the role this instance will play in the Splunk Enterprise deployment
+* `splunk.role`: the role this instance will play in the Splunk Enterprise deployment. (e.g. `splunk_standalone`)
 * `splunk.build_location`: URL to dynamically fetch the Splunk Enterprise build and install it at run time
-* `splunk.build_remote_src`: this wll be `true` when `splunk.build_location` above is a URL
+* `splunk.build_remote_src`: this wll be `true` when `splunk.build_location` above is a URL (e.g. `http://`, `https://`, or `file://`), and `false` if it is a path. Note: if a path is provided, the tgz, msi, deb, or rpm file that it points to must be in the container.
```

Comment:
super NIT, but you're missing an `i` in `will`. Also I think we discussed last time about changing the wording for `build_location` to path since the source could be a local path.